presto-main/src/main/java/com/facebook/presto/sql/Serialization.java

```diff
@@ -37,6 +37,9 @@
 
 public final class Serialization
 {
+    private static final char VARIABLE_TYPE_OPEN_BRACKET = '<';
```

Comment:
Angle bracket feels quite strange here. In general I prefer to fix it so "()" never exists in variable names. But this for the most part is not visible to end users so I won't be too picky about it. Please add a comment to explain why.